lib/react_client.dart

```diff
@@ -245,40 +318,10 @@ ReactComponentFactory _registerComponent(ComponentFactory componentFactory, [Ite
     )])
   ]);
```

Comment:
we probably get some sort of JS interop caching advantage out of explicitly calling `createFactory` (I didn't dive down into the `createElement` implementation to find a root cause)